code/src/components/Labyrinth.js

```diff
@@ -0,0 +1,210 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { navigateGame } from "../reducer/game";
+import { keyframes } from "styled-components";
+import Confetti from "./Confetti";
+import uniqid from "uniqid";
+import BG2 from "./img/bg2.png";
+import BG3 from "./img/bg3.png";
+import BG4 from "./img/bg4.png";
+import BG5 from "./img/bg5.png";
+import BG6 from "./img/bg6.png";
+import BG7 from "./img/bg7.png";
+import BG8 from "./img/bg8.png";
+import arrow from "./img/arrow.svg";
+import navigator from "./img/navigator.svg";
+
+import {
+  Background,
+  GameText,
+  ButtonController,
+  Btn,
+  MiddeBtn,
+  Modal,
+  HistoryArrowContainer,
+} from "./Styling";
+import Loadingspinner from "./Loadingspinner";
+import styled from "styled-components/macro";
+
+const HistoryArrow = styled.img`
+  transform: rotate(${(props) => props.rotate});
+`;
+
+const Navigator = styled.img`
+  transform: scale(0.2);
+  position: absolute;
+`;
+
+const Labyrinth = () => {
+  const [select, setSelect] = useState([]);
+  const [prevStep, setPrevStep] = useState("");
+  const dispatch = useDispatch();
+  const items = useSelector((store) => store.game.items);
+
+  document.addEventListener("keydown", (e) => {
+    if (e.key === "ArrowRight") {
+      dispatch(navigateGame("move", "East"));
+      setPrevStep("East");
+    } else if (e.key === "ArrowLeft") {
+      dispatch(navigateGame("move", "West"));
+      setPrevStep("West");
+    } else if (e.key === "ArrowDown") {
+      dispatch(navigateGame("move", "South"));
+      setPrevStep("South");
+    } else if (e.key === "ArrowUp") {
+      dispatch(navigateGame("move", "North"));
+      setPrevStep("North");
+    }
+
+    setTimeout(() => setPrevStep(""), 1500);
+  });
+
+  const onNavigate = (type, direction) => {
+    //STORE HISTORY, SELECTED STEPS
+    setSelect((prev) => [...prev, direction]);
+    // STORE LAST STEP
+    setPrevStep(direction);
+    /////////////////////////////////////
+    dispatch(navigateGame(type, direction));
+
+    setTimeout(() => setPrevStep(""), 1500);
+  };
+
+  const changeDirectionToIcon = (way) => {
+    if (way === "North") {
+      return <p>N</p>;
+    } else if (way === "South") {
+      return <p>S</p>;
+    } else if (way === "East") {
+      return <p>E</p>;
+    } else if (way === "West") {
+      return <p>W</p>;
+    }
+  };
+
+  const arrowStyle = (way) => {
+    if (way === "North") {
+      return "1/2";
+    } else if (way === "South") {
+      return "3/2";
+    } else if (way === "East") {
+      return "2/3";
+    } else if (way === "West") {
+      return "2/1";
+    }
+  };
+
```

Comment:
We didn't get the arrowStyle variable what the numbers 1/2, 3/2, 2/3 and 2/1  represents?